lib/logstash/codecs/fluent.rb

```diff
@@ -40,7 +40,8 @@ def decode(data, &block)
   end # def decode
 
   def encode(event)
-    tag = event.get("tags") || "log"
+   tag = event.get("tags") || "log"
```

Comment:
please remove this indentation changes, this will make the PR less noisy and more concise.